pkg/lib/patterns.js

```diff
@@ -378,7 +378,7 @@ $(document).ready(setup_sliders);
 // by default, tooltip is attached to "this" element; can be attached to
 // another one with setting "tooltip_element" (for <a> links)
 $.fn.update_privileged = function update_privileged(perm, denied_message, placement, tooltip_element) {
-    var allowed = (perm.allowed !== false);
+    var allowed = !!perm.allowed;
```

Comment:
Nice! This is exactly issue #12242, so I think this commit can claim "Fixes #". So now the overview page should have "Restart" disabled for a while until the root bridge is on? (That's a good change)

Reply:
Yes.  (But I can't see any visual difference between enabled and disabled with the new Overview...)